osconfig_tests/test_suites/package_management/package_management_test_data.go

```diff
@@ -146,26 +176,39 @@ func addPackageInstallRemovalTest(pkgTestSetup []*packageManagementTestSetup) []
 	for _, pkgManager := range pkgManagers {
 		var oc *osconfigpb.OsConfig
 		var assign *osconfigpb.Assignment
-		var instaneName, ss, vs string
+		var instanceName, image, ss, vs string
 
 		switch pkgManager {
 		case "apt":
-			instaneName = fmt.Sprintf("%s-%s", filepath.Base(debianImage), testName)
+			image = debianImage
+			instanceName = fmt.Sprintf("%s-%s", filepath.Base(image), testName)
 			pkg := osconfigserver.BuildPackage("cowsay")
 			installPkg := []*osconfigpb.Package{pkg}
 			pkg = osconfigserver.BuildPackage("cowsay")
 			removePkg := []*osconfigpb.Package{pkg}
-			oc = osconfigserver.BuildOsConfig(testName, desc, osconfigserver.BuildAptPackageConfig(installPkg, removePkg, nil), nil, nil, nil, nil)
-			assign = osconfigserver.BuildAssignment(testName, desc, osconfigserver.BuildInstanceFilterExpression(instaneName), []string{fmt.Sprintf("projects/%s/osConfigs/%s", testProjectID, oc.Name)})
-			ss = fmt.Sprintf("%s\nwhile true; do /usr/bin/dpkg-query -s wget | sudo tee /dev/ttyS0; sleep 5; done", utils.InstallOSConfigDeb)
+			oc = osconfigserver.BuildOsConfig(instanceName, desc, osconfigserver.BuildAptPackageConfig(installPkg, removePkg, nil), nil, nil, nil, nil)
+			assign = osconfigserver.BuildAssignment(instanceName, desc, osconfigserver.BuildInstanceFilterExpression(instanceName), []string{fmt.Sprintf("projects/%s/osConfigs/%s", testProjectID, oc.Name)})
+			ss = fmt.Sprintf("%s\nsudo apt-get -y install cowsay\nwhile true; do /usr/bin/dpkg-query -s cowsay | sudo tee /dev/ttyS0; sleep 5; done", utils.InstallOSConfigDeb)
 			vs = fmt.Sprintf("package 'cowsay' is not installed")
 			break
+		case "yum":
+			image = centosImage
+			instanceName = fmt.Sprintf("%s-%s", filepath.Base(image), testName)
+			pkg := osconfigserver.BuildPackage("cowsay")
+			installPkg := []*osconfigpb.Package{pkg}
+			pkg = osconfigserver.BuildPackage("cowsay")
+			removePkg := []*osconfigpb.Package{pkg}
```

Comment:
consider reducing this to 
```golang
installPkg := []*osconfigpb.Package{osconfigserver.BuildPackage("cowsay")}
removePkg := []*osconfigpb.Package{osconfigserver.BuildPackage("cowsay")}
```

The rational is that reusing `pkg` in this context is a bit confusing. It's assigned twice but each time to almost the same struct. And it's also unclear if `pkg` will be used again, so more cognitive load when reading the code.

Reply:
done.